build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld

```diff
@@ -285,56 +285,8 @@ SECTIONS
 		*(.BTTRACE)
 		__btrace_end__ = .;
 	} > BTRACE
-
-}
-
-SECTIONS
-{
-	.rom.text : { } > KM4_IROM
-	.rom.rodata : { } > KM4_DROM
-	.hal.rom.bss : { } > KM4_ROMBSS_RAM_COM
-	.hal.s_rom.bss : { } > KM4_ROMBSS_RAM_S
-
-	/* image3: security image start */
-	.gnu.sgstubs :
-	{
-		__ram_image3_nsc_start__ = .;
-		. = ALIGN(16);
-	} > KM4_BD_RAM_NSC
-
-	.ram_image3.entry :
-	{
-		KEEP(*(.image3.nsc_entry.text*))
-		. = ALIGN(16);
-	} > KM4_BD_RAM_ENTRY
-
-	.ram_image3.text :
-	{
-		__ram_image3_text_start__ = .;
-		*(.text*)
-		*(.rodata*)
-		. = ALIGN(16);
-	} > KM4_BD_RAM_S
-
-	.ram_image3.data :
-	{
-		__ram_image3_data_start__ = .;
-		*(.data*)
-		__ram_image3_data_end__ = .;
-
-		. = ALIGN(16);
-		__ram_image3_text_end__ = .;
-	} > KM4_BD_RAM_S
-
-	.ram_image3.bss :
-	{
-		__image3_bss_start__ = .;
-		*(.bss*)
-		*(COMMON)
-		__image3_bss_end__ = .;
-        end = .;
-	} > KM4_BD_RAM_S
-
+	__StackLimit = ABSOLUTE(ORIGIN(KM4_BD_RAM) + LENGTH(KM4_BD_RAM));
+	__PsramStackLimit = ABSOLUTE(ORIGIN(KM4_BD_PSRAM_NS_NTZ) + LENGTH(KM4_BD_PSRAM_NS_NTZ));

```

Comment:
Since PSRAM size is both KM4_BD_PSRAM_NS_NTZ & KR4_BD_PSRAM
Should the stack limit not include KR4_BD_PSRAM for LENGTH ?

Reply:
__PsramStackLimit if for available free heap calculation.
It to get the end point of KM4_BD_PSRAM_NS_NTZ, through the end point - ebss we can obtain the total free PSRAM available.
KR4_BD_PSRAM is another in-use region, it should not be included.

Reply:
@ziyik , As we discussed before, TP1x using 3M PSRAM for KM4
PSRAM_NS (rwx)  :       ORIGIN = 0x02000000, LENGTH = 0x02300000 - 0x02000000   /* PSRAM_NS:      3M */
But for the TP1e, there are setting to 1.8M
KM4_BD_PSRAM_NS_NTZ (rwx) :             ORIGIN = 0x60000020, LENGTH = 0x601D0000 - 0x60000020   /* KM4 MAIN RAM NS TRUSTZONE DIS: 1856k */

Thapa said it is not enough to use loadable app configuration. because loadable app kernal size is about 1.2M.
So, isn't it possible to rearrange the PSRAM size with KM4 and KR4 ?

Reply:
The expansion of PSRAM will not be done in this PR.
It will be included in a new PR with XIP modification.